Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -457,6 +458,7 @@ namespace PhysX
         AZ::Data::AssetBus::Handler::BusDisconnect();
         m_nonUniformScaleChangedHandler.Disconnect();
         AzFramework::BoundsRequestBus::Handler::BusDisconnect();
+        AzToolsFramework::EditorComponentSelectionRequestsBus::Handler::BusDisconnect();
```

Comment:
uber nit: It's always good to connect/disconnect in the reverse order so I'd move this just above the `BoundRequestBus` disconnect

Reply:
good spot! I thought I did but misplaced it by a line :P